### ISSUE TITLE
[Bug] Fog Configs for bclib don't work when BetterNether is installed

### ISSUE THREAD

**Issue by Orion113** [quiqueck/BetterNether#23]
### What happened?

An issue was raised on the bclib github recently about incompatibility with mods that affect fog rendering, which was addressed by adding config options for the custom fog renderer.

Those options work perfectly when bclib is installed by itself, but when it is installed with BetterNether, the config options no longer seem to have any affect in dimensions other than the overworld. The custom fog renders regardless of any other fog-affecting mods installed.

bclib (and other mods listed below) without betternether. Custom Fog Rendering and Nether Thick Fog both set to true:
![bclib only-config true](https://user-images.githubusercontent.com/108201263/175794544-93e76c75-9e3d-4b49-8484-cda174bd46a5.PNG)

bclib (and other mods listed below) without betternether. Custom Fog Rendering and Nether Thick Fog both set to false:
![bclib only-config false](https://user-images.githubusercontent.com/108201263/175794552-b263e5df-4f86-4db5-9d30-5b73a8c31845.PNG)

bclib (and other mods listed below) and betternether. Custom Fog Rendering and Nether Thick Fog both set to true:
![bclib and better nether-config true](https://user-images.githubusercontent.com/108201263/175794587-782f3500-5ed7-41f3-a7e1-ba88a27a4c49.PNG)

bclib (and other mods listed below) and betternether. Custom Fog Rendering and Nether Thick Fog both set to false:
![bclib and better nether-config false](https://user-images.githubusercontent.com/108201263/175794590-b5fcf58c-bbd4-4882-81c8-3fa9d10d2dae.PNG)

Tested in both the nether and end. Only mods installed are in the list below: fabric, mod menu for configs, nofog to test.

I also tried adjusting BetterNether's Fog Distance slider in its own config, and confirmed that the fog that is rendering responds to that slider.

### BetterNether

6.1.2

### BCLib

1.4.7

### Fabric API

0.56

### Fabric Loader

0.14.8

### Minecraft

1.18.2

### Other Mods

```shell
fabric-api-0.56.1+1.18.2
modmenu-3.2.2
NoFog-1.2.2_1.16.5-1.18.2-fabric
```

**Comment by Orion113** [comment]
Out of curiosity, I just went and tested this with the other BetterX mods. The config works fine with Better End and Eden Ring.

**Comment by quiqueck** [comment]
Thanks for reporting. BN is probably not (yet) aware of the BCLib config options.

**Comment by quiqueck** [comment]
Fixed for the next 1.19 Release

**Comment by quiqueck** [comment]
Fixed (for 1.19) in 7.0.4

**Comment by Orion113** [comment]
Awesome, so glad to hear it. Hope it will even make its way back to 1.18.2 eventually.
